data/governments.txt

```diff
@@ -505,6 +506,18 @@ government "Hai (Unfettered)"
 	"hostile hail" "hostile unfettered"
 	"hostile disabled hail" "hostile disabled unfettered"
 
+government "Hai (Unfettered) (Wanderer Tribute)"
+	swizzle 4
+	"display name" "Hai (Unfettered)"
+	"crew attack" 1.4
+	"crew defense" 2.2
+	
+	"player reputation" 0
+	"friendly hail" "friendly unfettered"
+	"friendly disabled hail" "friendly disabled unfettered"
+	"hostile hail" "hostile unfettered"
+	"hostile disabled hail" "hostile disabled unfettered"
+
```

Comment:
```suggestion
```
a new government is smth that should be avoided if possible, and here its totally not unavoidable, I think